web/src/message_lists.ts

```diff
@@ -8,8 +9,10 @@ import * as ui_util from "./ui_util";
 // TODO(typescript): Move this to message_list_view when it's
 // converted to typescript.
 type MessageListView = {
+    message_containers: Map<number, MessageContainer>;
     update_recipient_bar_background_color: () => void;
     rerender_messages: (messages: Message[], message_content_edited?: boolean) => void;
+    _get_msg_timestring: (message_container: MessageContainer) => string | undefined;
```

Comment:
`_get_msg_timestring` is clearly named as internal helper function for that module. I think rather than exporting it from `message_list_view.ts`, we should just move the function into `message_list_tooltips` with a more natural name; it doesn't seem to have other callers, and it should probably be called something like `get_last_edit_time`.

The merge conflict is from refactoring in `message_list_view.js` that fixed it to just take a `Message` object as a parameter, part of preparation for the `message_list_view` typescript migration.